geode-core/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableClientBug39997DUnitTest.java

```diff
@@ -51,7 +50,6 @@ public final void postTearDownCacheTestCase() {
 
   @Test
```

Comment:
Please rename this test to the current naming conventions and eliminate the bug number.

Reply:
Sure.